sklearn/metrics/classification.py

```diff
@@ -42,6 +42,17 @@
 from ..exceptions import UndefinedMetricWarning
 
 
+def _check_zero_division(zero_division):
+    if isinstance(zero_division, str):
+        if zero_division != "warn":
+            raise ValueError('zero_division must be one of ["warn", 0, 1]')
```

Comment:
Might as well add `f'Got {zero_division!r}'`

Reply:
I tried this but f-strings are not supported by all python versions we're testing with. So added with `.format`